setup.py

```diff
@@ -93,7 +93,7 @@ def find_version(*paths):
         'importlib_metadata;python_version<"3.9"'
     ],
     setup_requires=[
-        'pytest-runner~=4.0',
+        'pytest-runner~=5.3',
```

Comment:
If we're bumping this why not to v6? (the current)

Reply:
https://pypi.org/project/pytest-runner/ shows 5.3.1 as latest

Reply:
aah wait.. we can remove it from there. .. one sec

Reply:
Oh pytest runner different to pytest.

```
It is recommended that you:

    Remove 'pytest-runner' from your setup_requires, preferably removing the setup_requires option.
    Remove 'pytest' and any other testing requirements from tests_require, preferably removing the tests_requires option.
    Select a tool to bootstrap and then run tests such as tox.
```